package.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
the package.json in a node project allows to define a number of properties of the project; this one is fairly minimal, and helps clarify that:
* it requires node 20+ (since we're using the brand new toSorted() method on arrays)
* you need jsdom to run it
* you need mocha to develop it (used as test harness)
* it allows to use `npm run test` to launch the test suite (used in the github action above, and also useful to test locally)